azure-application-insights-spring-boot-starter/README.md

```diff
@@ -129,6 +129,26 @@ azure.application-insights.logger.type=console
 # Logging level [all, trace, info, warn, error, off]. Default value: error.
 azure.application-insights.logger.level=error
 
+# Enable/Disable developer mode, all telemetry will be sent immediately without batching. Significantly affects performance and should be used only in developer environment. Default value: false.
+azure.application-insights.channel.in-process.developer-mode=false
+# Endpoint address, Default value: none.
```

Comment:
I think default this is little bit misleading to users. If the user doesn't specify the endpoint address the default endpoint would be Application Insights backend service which is "https://dc.services.visualstudio.com/v2/track". We should mention something that - if no value is specified traffic would sendto application insights default backend service.

Reply:
This was the only property I was not quite sure. Don't you think this is something internal and should not be changed by simply using property value?

Reply:
@gavlyukovskiy well, if the user wants to send the data to some different end point he or she should have flexibility to do so. What do you think? We can keep this as it is for now and change based on others feedback here. But I think there is no harm personally

Reply:
Added default value.